service/pointinpolygon.js

```diff
@@ -2,14 +2,51 @@ const logger = require( 'pelias-logger' ).get( 'pointinpolygon' );
 const request = require('request');
 const _ = require('lodash');
 
+function sanitizeUrl(requestUrl) {
+  return requestUrl.replace(/^(.+)\/.+\/.+$/, (match, base) => {
+    return `${base}/[removed]/[removed]`;
+  });
+}
```

Comment:
There are tests [here](https://github.com/pelias/api/pull/852/files#diff-f1db4ccb690ad5f0b8838fc6aece9abbR213) and [here](https://github.com/pelias/api/pull/852/files#diff-f1db4ccb690ad5f0b8838fc6aece9abbR298).